apps/examples/testcase/le_tc/network/tc_net_getpeername.c

```diff
@@ -125,12 +124,13 @@ static void tc_net_getpeername_close_n(void)
 	sock = socket(AF_INET, SOCK_STREAM, 0);
 	int len = sizeof(struct sockaddr);
 	struct sockaddr foo;
-	close(sock);
+
```

Comment:
this is negative case.
I think closing sock is getting invalid sock intentionally.
please check.